code/game/objects/items/food/pastries.dm

```diff
@@ -751,19 +752,28 @@
 			desc = "A fat stack of fluffy pancakes!"
 		if(7 to 9)
 			desc = "A grand tower of fluffy, delicious pancakes!"
-		if(PANCAKE_MAX_STACK to INFINITY)
+		if(PANCAKE_MAX_STACK)
 			desc = "A massive towering spire of fluffy, delicious pancakes. It looks like it could tumble over!"
+		if(11 to 99)
+			desc = "A completely mind bogglingly tremendous amount of fluffy and wonderful pancakes! It towers over the realm of the mortals, watching the petty squabbles of day to day from it's grandiose pile of battery goodness."
```

Comment:
```suggestion
			desc = "A completely mindbogglingly tremendous amount of fluffy and wonderful pancakes! It towers over the realm of the mortals, watching the petty squabbles of day to day from its grandiose pile of battery goodness."
```

Reply:
I think mind-bogglingly should be hyphenated.

Reply:
firefox's spellcheck and every dictionary i can find would seem to disagree

Reply:
https://en.wiktionary.org/wiki/mind-boggling#Alternative_forms
As everyone knows, US forms are wrong.

Reply:
either way its not mind bogglingly